src/main/java/com/github/jcustenborder/kafka/connect/redis/ByteString.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2014-2020 project44. All rights reserved.
```

Comment:
Is this correct to have on an open source fork? I don't really know.

Reply:
Yeah me neither